modules/ROOT/pages/6.8-release-notes.adoc

```diff
@@ -276,8 +276,20 @@ waiting for confirmation on these new feature tickets.
 === The dialog size was not updated when the size argument was changed when redialling a dialog.
 // #TINY-10209
 
-=== Custom block element wasn't considered block element in some cases.
+=== Custom block element's where not considered block elements in some cases.
 // #TINY-10139
+Before, if a user copied and pasted a block element that was inside a custom element like the example config below, the editor would make an duplicate copy of the outer wrapper for the custom block element when pasting the content back. This happened because the function didn't check if the element is a wrapper element.
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  extended_valid_elements: 'mycustomblock[id],mycustominline',
+  custom_elements: 'mycustomblock,~mycustominline' // Notice the ~ prefix to force a span element for the element
+});
+----
+
+{productname} 6.8, now, checks if the element is a wrapper element, so it is also considers elements contained in custom element tags.
```

Comment:
the problem here wasn't that there were no check but that the check didn't consider the custom elements

Reply:
```suggestion
{productname} 6.8, now also considers **custom elements** as part of its process when checking if the element is a wrapper element.
```

Reply:
@lorenzo-pomili Please advise if you are happy with the above suggestion.

Reply:
it works for me